docs/conf.py

```diff
@@ -25,7 +25,7 @@
 
 source_suffix = {".rst": "restructuredtext"}
 templates_path = ["_templates"]
-html_static_path = ["assets"]
+html_static_path = ["assets", "_static"]
```

Comment:
Does it have to be a new directory? I don't have a strong opinion, but it seems like it fits well enough under `assets`, and one place for files is a bit simpler than two.

Reply:
This is the part I'm not sure about.. I also think putting the file in "assets" folder makes the most sense, but I did not get that to work because after compilation, the `versions.json` file ends up at "_static" folder, and when fetching file from "assets", the file is not there..